packages/arcgis/src/geojson.js

```diff
@@ -59,42 +59,42 @@ export const geojsonToArcGIS = (geojson, idAttribute) => {
     case 'Point':
       result.x = geojson.coordinates[0];
       result.y = geojson.coordinates[1];
-      if (geojson.coordinates[2]) {
+      if (geojson.coordinates[2] || geojson.coordinates[2] === 0) {
```

Comment:
Maybe more concise to just do `if (geojson.coordinates[2] !== undefined)`.

Reply:
@rgwozdz and i are on the same wavelength.

i was going to suggest `!= null`. (ref: https://stackoverflow.com/a/15992131/3019940)

Reply:
Even better 👍 

Thanks, changing now

Reply:
Addressed by f901c5c